app/src/main/java/org/hackillinois/android/view/LoginActivity.kt

```diff
@@ -0,0 +1,108 @@
+package org.hackillinois.android.view
+
+import android.content.Context
+import android.content.Intent
+import android.net.Uri
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import android.util.Log
+import android.widget.Button
+import org.hackillinois.android.App.getAPI
+import org.hackillinois.android.R
+import org.hackillinois.android.model.Code
+import org.hackillinois.android.model.JWT
+import org.hackillinois.android.model.User
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
+
+class LoginActivity : AppCompatActivity() {
+    private lateinit var attendeeLogin: Button
+    private lateinit var staffLogin: Button
+    private lateinit var recruiterLogin: Button
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_login)
+
+        attendeeLogin = findViewById(R.id.attendeeLoginBtn)
+        staffLogin = findViewById(R.id.staffLoginBtn)
+        recruiterLogin = findViewById(R.id.recruiterLoginBtn)
```

Comment:
With the `kotlinx` android extensions, you don't need to use `findViewById`. Instead, when using the buttons below these lines, just call the id, such as `attendeeLoginBtn.setOnClickListener { ... }`. Android studio should be able to autoimport this, but if not you can look at some of the other Kotlin fragments to see the import.